atom-publisher-lib/build.sbt

```diff
@@ -18,7 +18,14 @@ startDynamoDBLocal <<= startDynamoDBLocal.dependsOn(compile in Test)
 test in Test <<= (test in Test).dependsOn(startDynamoDBLocal)
 testOptions in Test <+= dynamoDBLocalTestCleanup
 
+dependencyOverrides += "org.apache.thrift" % "libthrift" % "0.9.1"
```

Comment:
I'm tempted to force the upgrade here since there's a limited number of projects using atom-maker.

Worth trying in atom-workshop and seeing how painful it is? Up to you